css/comp-page.css

```diff
@@ -72,6 +88,7 @@
 }
 
 .comp-disp {
```

Comment:
Have consistency with class names. Somewhere it's 'component-' and here (and following classes) it's 'comp-'